core/include/kernel/dt.h

```diff
@@ -69,6 +69,18 @@ enum dt_driver_type {
 	DT_DRIVER_RSTCTRL,
 };
 
+/*
+ * DT_MAP_FROM_NODE_STATUS: Uses status properties from device tree to
+ * determine mapping.
+ * DT_MAP_SECURE: Force mapping for device to be secure.
+ * DT_MAP_NON_SECURE: Force mapping for device to be non-secure.
+ */
+enum dt_map_dev_directive {
+	DT_MAP_FROM_NODE_STATUS,
```

Comment:
Was thinking should this be `DT_MAP_AUTO` or so.

Reply:
Yes, that's a good name.

Reply:
I wait a bit if there are other comments and then go with this (needs a change in quite a many places :sweat:)

Reply:
FWIW... `DT_MAP_AUTO` sounds good to me too